python/cudf/cudf/core/column/timedelta.py

```diff
@@ -0,0 +1,565 @@
+                ).binary_operator(
+                    "floordiv",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["h"], "ns")
+                    ),
+                ),
+                "minutes": self.binary_operator(
+                    "mod",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["h"], "ns")
+                    ),
+                ).binary_operator(
+                    "floordiv",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["m"], "ns")
+                    ),
+                ),
+                "seconds": self.binary_operator(
+                    "mod",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["m"], "ns")
+                    ),
+                ).binary_operator(
+                    "floordiv",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["s"], "ns")
+                    ),
+                ),
+                "milliseconds": self.binary_operator(
+                    "mod",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["s"], "ns")
+                    ),
+                ).binary_operator(
+                    "floordiv",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["ms"], "ns")
+                    ),
+                ),
+                "microseconds": self.binary_operator(
+                    "mod",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["ms"], "ns")
+                    ),
+                ).binary_operator(
+                    "floordiv",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["us"], "ns")
+                    ),
+                ),
+                "nanoseconds": self.binary_operator(
+                    "mod",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["us"], "ns")
+                    ),
+                ).binary_operator(
+                    "floordiv",
+                    as_scalar(
+                        np.timedelta64(_numpy_to_pandas_conversion["ns"], "ns")
+                    ),
+                ),
+            },
+            index=index,
+        )
+
+    @property
+    def days(self):
+        return self.binary_operator(
+            "floordiv",
+            as_scalar(np.timedelta64(_numpy_to_pandas_conversion["D"], "ns")),
+        )
+
+    @property
+    def seconds(self):
+        # This property must return the number of seconds (>= 0 and
+        # less than 1 day) for each element, hence first performing
+        # mod operation to remove the number of days and then performing
+        # division operation to extract the number of seconds.
```

Comment:
Should have a docstring to clarify that it's the number of seconds less than 1 day. Leave implementation details as a comment.